pkg/galera/gtid_test.go

```diff
@@ -0,0 +1,92 @@
+package galera_test
+
+import (
+	"testing"
+
+	"github.com/google/go-cmp/cmp"
+	"github.com/mariadb-operator/agent/pkg/galera"
+)
+
+func TestGTIDUnmarshal(t *testing.T) {
+	var gtid galera.GTID
+	err := gtid.UnmarshalText([]byte("0-1`-2"))
```

Comment:
```suggestion
	err := gtid.UnmarshalText([]byte("0-1-2"))
```
Was the extra quote intended? It's making the tests fail

Reply:
Good catch! It was of course not intended aha